geolocation/src/definitions.ts

```diff
@@ -52,7 +52,7 @@ export interface GeolocationPlugin {
    * @since 1.0.0
    */
   watchPosition(
-    options: PositionOptions,
+    options: WatchPositionOptions,
```

Comment:
I'm not sure I need to add a new interface here.

Reply:
Not really, you can just add the new options to the existing interface.